packages/datadog-instrumentations/src/winston.js

```diff
@@ -43,12 +43,12 @@ function wrapMethod (method, logCh) {
         if (patched.has(transport) || typeof transport.log !== 'function') continue
 
         const log = transport.log
-        transport.log = function wrappedLog (level, msg, meta, callback) {
+        transport.log = shimmer.wrapFunction(log, log => function wrappedLog (level, msg, meta, callback) {
```

Comment:
shouldn't this be 
```javascript
shimmer.wrap(transport, 'log', ...)
```
instead ?

Reply:
It probably always should have been `shimmer.wrap()` yeah. I can change this one.

Reply:
Done.